onnxruntime/core/session/ort_apis.h

```diff
@@ -244,7 +244,9 @@ ORT_API_STATUS_IMPL(SetGlobalSpinControl, _Inout_ OrtThreadingOptions* tp_option
 ORT_API_STATUS_IMPL(AddInitializer, _Inout_ OrtSessionOptions* options, _In_ const char* name,
                     _In_ const OrtValue* val);
 
-ORT_API_STATUS_IMPL(OrtSessionOptionsAppendExecutionProvider_CUDA,
-                    _In_ OrtSessionOptions* options, _In_ OrtCUDAProviderOptions* cuda_options);
+ORT_API_STATUS_IMPL(SessionOptionsAppendExecutionProvider_CUDA,
```

Comment:
After this PR we'll have 3 variations of openvino APIs - 2 in the factory header and one here. I guess we plan to consolidate these in one place before this PR is merged or add appropriate documentation to inform our users?

Reply:
Yeah, I'll talk to Hector about it. Ideally we'll just have the C API one, and not have the dynamically exported functions at all.